db/migrate/20180125095124_add_preferred_gender_to_families.rb

```diff
@@ -0,0 +1,5 @@
+class AddPreferredGenderToFamilies < ActiveRecord::Migration[5.1]
+  def change
+    add_column :families, :preferred_gender, :string
```

Comment:
Since you are only just starting, I would add this line to the previous migration.

In bigger project, trying to keep the migration folders in an organised state is hard. Thus ^